Yosemite/Yosemite/Model/Storage/Refund+ReadOnlyConvertible.swift

```diff
@@ -16,6 +16,7 @@ extension Storage.Refund: ReadOnlyConvertible {
         amount = fullRefund.amount
         reason = fullRefund.reason
         byUserID = fullRefund.refundedByUserID
+        supportShippingRefunds = fullRefund.shippingLines != nil
```

Comment:
nit: what about also having this property in the Networking layer and logic there so that we can test this property along with other properties? it might be hard to find this property here in the future 🙂

Reply:
I didn't add this to the networking entity because of two reasons:
1. I thought networking entities should be a 1-1 match with API objects.
2. I plan to delete this once we add support to something like this https://github.com/woocommerce/woocommerce-ios/issues/3200